odl/tomo/geometry/parallel.py

```diff
@@ -36,59 +37,62 @@ class ParallelGeometry(Geometry):
     detector.
     """
 
-    def __init__(self, ndim, apart, detector, det_init_pos):
+    def __init__(self, ndim, apart, detector, det_pos_init):
         """Initialize a new instance.
 
         Parameters
         ----------
         ndim : {2, 3}
             Number of dimensions of this geometry, i.e. dimensionality
-            of the physical space in which this geometry is embedded
+            of the physical space in which this geometry is embedded.
         apart : `RectPartition`
-            Partition of the angle set
+            Partition of the angle set.
         detector : `Detector`
-            The detector to use in this geometry
-        det_init_pos : `array-like`
-            Initial position of the detector reference point
+            The detector to use in this geometry.
+        det_pos_init : `array-like`
+            Initial position of the detector reference point.
         """
         super().__init__(ndim, apart, detector)
 
         if self.ndim not in (2, 3):
-            raise ValueError('number of dimensions is {}, expected 2 or 3'
-                             ''.format(ndim))
+            raise ValueError('`ndim` must be 2 or 3, got {}'.format(ndim))
 
-        self._det_init_pos = np.asarray(det_init_pos, dtype='float64')
-        if self._det_init_pos.shape != (self.ndim,):
-            raise ValueError('initial detector position has shape {}, '
-                             'expected ({},)'
-                             ''.format(self._det_init_pos.shape, self.ndim))
+        self.__det_pos_init = np.asarray(det_pos_init, dtype='float64')
+        if self.det_pos_init.shape != (self.ndim,):
+            raise ValueError('`det_pos_init` must have shape ({},), got {}'
+                             ''.format(self.ndim, self.det_pos_init.shape))
+
+    @property
+    def det_pos_init(self):
```

Comment:
There was already an inconsistency with `src_to_det_init` in the divergent beam geometries, and I changed it in favor of this one. Reason: `det_position` and `det_pos_init` will show up close to each other in autocomplete lists.